cpp/README.md

```diff
@@ -1,118 +1,119 @@
 # Ice for C++
 
-[Getting started] | [Examples] | [NuGet packages] | [Documentation] | [Building from source]
+[Examples] | [Ice Manual] | [Building from source]
```

Comment:
I think we should say documentation. It's fine that we call it that but I think when linking to it we should avoid the term manual.